reana_client/cli/workflow.py

```diff
@@ -526,8 +540,15 @@ def workflow_logs(ctx, workflow, access_token, json_format):
     help='REANA specifications file describing the workflow and '
          'context which REANA should execute.')
 @click.pass_context
-def workflow_validate(ctx, file):
-    """Validate given REANA specification file."""
+def workflow_validate(ctx, file):  # noqa: D301
+    """Validate workflow specification file.
+
+    The `validate` command permits to execute validation of workflow
```

Comment:
That may be a bit ambiguous, better say something like:

The `validate` command permits to check syntax and validate the `reana.yaml` workflow specification file.